### PR TITLE
[ICU] Upgrade ICU to 65.1

### PR DESCRIPTION
**Pull request by xuhongxu96**
Upgrade `ICU` to 65.1.

### CODE REVIEW

**Comment by xuhongxu96** [comment]
There are some compile issues. Pending.

**Comment by maksqwe** [comment]
@grdowns please take a look. The current ICU version 61 cannot be downloaded.

File: ports/icu/portfile.cmake [hunk 1]
```diff
@@ -4,15 +4,17 @@ endif()
 
 include(vcpkg_common_functions)
```
Comment (LilyWangL) [inline ports/icu/portfile.cmake]: include(vcpkg_common_functions) is deprecated. Please remove it. 

Thanks.

**Review by LilyWangL — CHANGES_REQUESTED** [review]

File: ports/icu/portfile.cmake (hunk 1 above)
Comment (xuhongxu96) [reply ports/icu/portfile.cmake]: Done.

**Review by PhoebeHui — APPROVED** [review]
LGTM, thanks for the PR!

**Review by LilyWangL — APPROVED** [review]

**Review by JackBoosY — APPROVED** [review]

**Comment by ras0219-msft** [comment]
Thanks for the update!

**Pull request merged**